pkg/controller/provider/container/ovirt/model.go

```diff
@@ -41,10 +42,12 @@ const (
 	USER_ADD_HOST    = 42
 	USER_UPDATE_HOST = 43
 	USER_REMOVE_HOST = 44
+	// Template
+	USER_ADD_VM_TEMPLATE                  = 48
+	USER_ADD_VM_TEMPLATE_FINISHED_SUCCESS = 51
 	// VM
 	USER_ADD_VM                           = 34
-	USER_ADD_VM_STARTED                   = 37
-	USER_ADD_VM_FINISHED_FAILED           = 52
+	USER_ADD_VM_FINISHED_SUCCESS          = 53
```

Comment:
USER_ADD_VM_STARTED not useful.
USER_ADD_VM generated for synchronous create.
USER_ADD_VM_TEMPLATE_FINISHED_SUCCESS generated when created asynchronous task succeeds.

Since we are gracefully handling (ignoring) events between USER_ADD_VM_STARTED and USER_ADD_VM_TEMPLATE_FINISHED_SUCCESS we can just trigger off of USER_ADD_VM_TEMPLATE_FINISHED_SUCCESS.